hindsight/io_modules/derived_stream/redshift/psv.lua

```diff
@@ -13,13 +13,13 @@ local string    = require "string"
 
 setfenv(1, M) -- Remove external access to contain everything in the module
 
-local esc_chars = { ["|"] = "\\|", ["\r"] = "\\r", ["\n"] = "\\n" }
+local esc_chars = { ["|"] = "\\|", ["\r"] = "\\r", ["\n"] = "\\n", ["\0"] = "0" }
```

Comment:
re: log, just a simple "last N observed strings containing NUL" heka plugin would do for me.  Just want some visibility into the problem.